src/AdminModule/Components/Sidebar/Sidebar.tsx

```diff
@@ -1,7 +1,312 @@
+import Divider from '@mui/material/Divider';
+import AutoAwesomeIcon from '@mui/icons-material/AutoAwesome';
+import IconButton from '@mui/material/IconButton';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
+import ListItem from '@mui/material/ListItem';
+import Modal from 'react-bootstrap/Modal';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import BeenhereIcon from '@mui/icons-material/Beenhere';
+import ListItemText from '@mui/material/ListItemText';
+import AppsIcon from '@mui/icons-material/Apps';
+import LockIcon from '@mui/icons-material/Lock';
+import { useNavigate } from 'react-router-dom';
+import ChangePassword from '../../../AuthModule/Components/ChangePassword/ChangePassword';
+import { Link } from 'react-router-dom';
+
+import { InsertLink } from '@mui/icons-material';
+const drawerWidth = 210;
+
+const openedMixin = (theme: Theme): CSSObject => ({
+  width: drawerWidth,
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.enteringScreen,
+  }),
+  overflowX: 'hidden',
+});
+
+const closedMixin = (theme: Theme): CSSObject => ({
+  transition: theme.transitions.create('width', {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  overflowX: 'hidden',
+  width: `calc(${theme.spacing(7)} + 1px)`,
+  [theme.breakpoints.up('sm')]: {
+    width: `calc(${theme.spacing(8)} + 1px)`,
+  },
+});
+
+const DrawerHeader = styled('div')(({ theme }) => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  backgroundColor:"#203FC7",
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+}));
+
+
+
+
+const Drawer = styled(MuiDrawer, { shouldForwardProp: (prop) => prop !== 'open' })(
+  ({ theme, open }) => ({
+    width: drawerWidth,
+    flexShrink: 0,
+    whiteSpace: 'nowrap',
+    boxSizing: 'border-box',
+    ...(open && {
+      ...openedMixin(theme),
+      '& .MuiDrawer-paper': openedMixin(theme),
+    }),
+    ...(!open && {
+      ...closedMixin(theme),
+      '& .MuiDrawer-paper': closedMixin(theme),
+    }),
+  }),
+);
+
 
 export default function Sidebar() {
+
+  const navigateTo=useNavigate();
+  const theme = useTheme();
+  const [open, setOpen] = React.useState(false);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+ 
+  const [show, setShow] = React.useState(false);
+  const handleShow = () => setShow(true);
+  const handleClose = () => setShow(false);
+ 
   return (
-    <div>Sidebar</div>
-  )
+    <>
+   < Modal show={show} onHide={handleClose}>
+        <Modal.Body>
+        <ChangePassword handleClose={handleClose}/>
+        </Modal.Body>
+      </Modal>
+
+  
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
```

Comment:
The CssBaseline component should be included in the top level component like the App component. it is similar to the CSS reset, so we don't need to include it in different places.